Sources/MapboxMaps/Foundation/Camera/BasicCameraAnimator.swift

```diff
@@ -33,23 +34,30 @@ public class BasicCameraAnimator: NSObject, CameraAnimator, CameraAnimatorInterf
         switch internalState {
         case let .running(transition), let .paused(transition):
             return transition
-        case .initial, .final:
+        case .delayed, .initial, .final:
             return nil
         }
     }
 
     /// A timer used to delay the start of an animation
-    private var delayedAnimationTimer: Timer?
+    internal var delayedAnimationTimer: Timer?
 
     /// The state from of the animator.
-    public var state: UIViewAnimatingState { propertyAnimator.state }
+    public var state: UIViewAnimatingState {
+        switch internalState {
+        case .delayed:
+            return .active
+        default:
+            return propertyAnimator.state
+        }
+    }
 
     private var internalState = InternalState.initial {
         didSet {
             switch (oldValue, internalState) {
-            case (.initial, .running), (.paused, .running):
+            case (.initial, .running), (.paused, .running), (.initial, .delayed), (.delayed, .running):
```

Comment:
add .paused -> .delayed
```suggestion
            case (.initial, .running), (.paused, .running), (.initial, .delayed), (.paused, .delayed):
```